src/store/components/AdminFulfillPage/index.tsx

```diff
@@ -122,6 +133,18 @@ const AdminFulfillPage: React.FC<AdminFulfillPageProps> = (props) => {
               ?.filter((order) => {
                 return order.status !== OrderStatus.CANCELLED;
               })
+              .sort((a, b) => {
+                const aName = `${a.user.firstName} ${a.user.lastName}`;
+                const bName = `${b.user.firstName} ${b.user.lastName}`;
+
+                if (aName < bName) {
+                  return -1;
+                }
+                if (aName > bName) {
+                  return 1;
+                }
+                return 0;
```

Comment:
why not use `String.prototype.localeCompare`?

Reply:
Good point, I swapped all string comparison instances with `localeCompare`. For some reason when I searched for JS string comparison this didn't pop up 🧐